core/src/main/scala/com/broadinstitute/dsp/DbReaderImplicits.scala

```diff
@@ -113,34 +113,30 @@ object DbReaderImplicits {
       case (id, name, cloudContextDb, location, cloudProvider) =>
         cloudProvider match {
           case CloudProvider.Azure =>
-            AzureCloudContext.fromString(cloudContextDb) match {
-              case Left(value) =>
-                throw new RuntimeException(
-                  s"${value} is not valid azure cloud context"
-                )
-              case Right(value) =>
-                KubernetesCluster(id, name, CloudContext.Azure(value), location)
-            }
+            throw new RuntimeException(
+              s"kubernetesCluster(${id}) is Azure cluster. This is impossible. Fix this in DB"
```

Comment:
For this error message, and the one below, I would suggest to make it a bit more actionable. We only expect GCP kubernetes cluster to be returned, but why? And what is the fix to be done in the DB?